nixos/doc/manual/configuration/profiles/perlless.section.md

```diff
@@ -0,0 +1,5 @@
+# Perlless {#sec-perlless}
+
+Render your system completely perlles (i.e. without the perl interpreter). This
```

Comment:
```suggestion
Render your system completely perl-less (i.e. without the perl interpreter). This
```

Reply:
No why are you doing this to me?

Reply:
choose between perlless or perl-less, but no perlles :P.

Reply:
The [nixpkgs manual](https://nixos.org/manual/nixpkgs/stable/#setup-hook-perl) refers to it either as `perl` (the command) or Perl (the language). It would be nice to be consistent.

Can you also add a disclaimer here about the `nixos-rebuild switch` limitation? This could be very surprising to people.